### PR TITLE
Ability to view a list of users assigned each role

### PR DESCRIPTION
**Pull request by k7hpn**

### CODE REVIEW

File: src/GRA.Controllers/MissionControl/RolesController.cs [hunk 1]
```diff
@@ -20,47 +20,53 @@ public class RolesController : Base.MCController
...
-                return RedirectToRoute(
-                    new
-                    {
-                        page = paginateModel.LastPage ?? 1
-                    });
+                return RedirectToRoute(new { page = paginateModel.LastPage ?? 1 });
             }
 
-            var viewModel = new RoleListViewModel
+            var viewmodel = new RoleListViewModel
```
Comment (madison-boman) [inline src/GRA.Controllers/MissionControl/RolesController.cs:60]: Why was the camelcase removed on this?

File: src/GRA.Web/Areas/MissionControl/Views/Roles/Detail.cshtml [hunk 2]
```diff
@@ -16,9 +16,20 @@
...
+            }
+            else
+            {
+                <button class="btn btn-default" disabled>No users in this role</button>
+            }
+
             <div class="row">
                 <div class="col-xs-6">
-                    <h3>Available Permissions</h3>
+                    <h3>Available Permissions (@Model.UnselectedPermissions?.Count())</h3>
```
Comment (madison-boman) [inline src/GRA.Web/Areas/MissionControl/Views/Roles/Detail.cshtml:32]: This should be an `h2` with the `class="h3"`.  With those changes the view looks the same. This is just an accessibility issue we should get in routine of.

File: src/GRA.Web/Areas/MissionControl/Views/Roles/Detail.cshtml [hunk 3]
```diff
@@ -31,7 +42,7 @@
                 </div>
 
                 <div class="col-xs-6">
-                    <h3>Has Permissions</h3>
+                    <h3>Has Permissions (@Model.SelectedPermissions?.Count())</h3>
```
Comment (madison-boman) [inline src/GRA.Web/Areas/MissionControl/Views/Roles/Detail.cshtml:45]: This should be an `h2` with the `class="h3"`.  With those changes the view looks the same. This is just an accessibility issue we should get in routine of.

**Review by madison-boman — COMMENTED** [review]
Looks and works great overall!

File: src/GRA.Controllers/MissionControl/RolesController.cs (hunk 1 above)
Comment (k7hpn) [reply src/GRA.Controllers/MissionControl/RolesController.cs]: I got lazy and tired of hitting the shift key ;)

File: src/GRA.Web/Areas/MissionControl/Views/Roles/Detail.cshtml (hunk 3 above)
Comment (k7hpn) [reply src/GRA.Web/Areas/MissionControl/Views/Roles/Detail.cshtml]: Good catches on these, I'll try to be more on top of this!

**Review by iafb — APPROVED** [review]

**Pull request merged**